modules/nw-scaling-ingress-controller.adoc

```diff
@@ -0,0 +1,93 @@
+// Module filename: nw-scaling-ingress-controller.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-ingress-controller-configuration-{context}"]
+= Scaling an ingress controller
+
+An ingress controller can be manually scaled up or down based on
```

Comment:
What's the convention for referring to API resources like this? For example:

* "ingress controller"
* `ingresscontroller`

Reply:
I've been using ```an `IngressController` resource``` for referring to a concrete API object and "an ingress controller" to refer to an ingress controller in abstract.

Reply:
I believe upstream takes the same approach that @Miciah describes. I will update the doc accordingly.